NuGet.config

```diff
@@ -6,8 +6,10 @@
   </config>
   <packageSources>
     <clear />
+    
     <add key="ProjectReunion internal" value="https://microsoft.pkgs.visualstudio.com/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" />
-    <add key="WindowsAppSDK Local" value="tools/nuget" />
+    <add key="WindowsAppSDK Local" value="tools/nuget" />    
+    <add key="localpackages" value="localpackages" />
```

Comment:
does this build (restore) without error on a fresh clone without a localpackages folder?  I've seen that fail before.

Reply:
There is a localpackages dir in the root of this repo. So it should be good.

Reply:
> There is a localpackages dir in the root of this repo. So it should be good.

right - there's a readme.md to gaurantee that